src/main/java/com/cdancy/jenkins/rest/features/JobsApi.java

```diff
@@ -194,4 +212,15 @@ String lastBuildTimestamp(@Nullable @PathParam("optionalFolderPath") @ParamParse
     ProgressiveText progressiveText(@Nullable @PathParam("optionalFolderPath") @ParamParser(OptionalFolderPathParser.class) String optionalFolderPath,
                                     @PathParam("name") String jobName,
                                     @QueryParam("start") int start);
+
+    @Named("jobs:build-number-progressive-text")
+    @Path("{optionalFolderPath}job/{name}/{number}/logText/progressiveText")
+    @Fallback(Fallbacks.NullOnNotFoundOr404.class)
+    @ResponseParser(OutputToProgressiveText.class)
+    @Consumes(MediaType.TEXT_PLAIN)
+    @GET
+    ProgressiveText buildNumberProgressiveText(@Nullable @PathParam("optionalFolderPath") @ParamParser(OptionalFolderPathParser.class) String optionalFolderPath,
```

Comment:
Lets just rename this to `progressiveText`.

Reply:
I am not sure I understand why this would be a progressive text. These are all permalinks that return simple strings.

Reply:
How about renaming it progressiveLogText()?

Reply:
Working on a new PR based on this one to clean things up and provide more tests. I'll tag you when it's ready.

Reply:
Basically I'd like to finally solve the RequestStatus issue and have 1 generic and common typed Object to use. I have something going but I'm not sure RequestStatus is the best name. Maybe JenkinsResponse? IDK. Up for any suggestions though.

Reply:
IDK. Maybe not. Maybe smaller more targeted versions is better. I keep going back and forth on this.

Reply:
So going with our current design and just doing it across the board and trying to simplify it a bit. Sorry for the rants and potential confusion while I was thinking out loud.